libretroshare/src/ft/ftserver.cc

```diff
@@ -1827,7 +1830,7 @@ int ftServer::handleIncoming()
  **********************************
  *********************************/
 
-void ftServer::receiveSearchResult(RsTurtleFTSearchResultItem *item)
```

Comment:
why this change of names? This function overloads the method in TurtleClientService. So it should not be changed! If you do, FT will just not work anymore.

Reply:
Looks at warning:
/libretroshare/src/ft/ftserver.h:101: warning:
'ftServer::receiveSearchResult' hides overloaded virtual function [-
Woverloaded-virtual]
/libretroshare/src/turtle/turtleclientservice.h:122: hidden overloaded
virtual function 'RsTurtleClientService::receiveSearchResult' declared
here: different number of parameters (3 vs 1)

Reply:
This is a relevant warning...

Reply:
yes I agree. The very weird thing here is that I'm pretty sure to have fixed this already, but apparently I did not. Gio told me the exact same thing weeks ago, and I thought I fixed it.
So: yes, change the same to ftServerReceiveSearchResults() and call this function in p3turtle.cc:1223  instead of             client->receiveSearchResult(ftsr);

Reply:
OK, but I test if it is a ftServer before.